css/style.css

```diff
@@ -0,0 +1,549 @@
+}
+.mobile-close-btn{
+    width: 28px;
+    margin-left: -10px;
+}
+
+.show{
+    display: block;
+}
+
+.no-scroll{
+    overflow: hidden;
+}
+
+/* holidays-list */
+.holidays-list{
+    max-width: 200px;
+}
+
+.holidays-list_item{
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 18px;
+    letter-spacing: 0.05em;
+    text-transform: uppercase;
+    padding-left: 40px;
+    position: relative;
+    padding-bottom: 25px;
+}
+
+.holidays-list_item-1::before{
+    content: url(/images/holiday-1.svg);
+    position: absolute;
+    left: -10px;
+    top: -6px;
+}
+
+.holidays-list_item-2::before{
+    content: url(/images/holiday-2.svg);
+    position: absolute;
+    left: -10px;
+    top: -6px;
+}
+
+.holidays-list_item-3::before{
+    content: url(/images/holiday-3.svg);
+    position: absolute;
+    left: -10px;
+    top: -6px;
+}
+
+.holidays-list_item-4::before{
+    content: url(/images/holiday-4.svg);
+    position: absolute;
+    left: -10px;
+    top: -6px;
+}
+
+/* banner */
+
+.banner{
+    color: #FFFFFF;
+    /* height: 150px; */
+}
+
+.banner-wrapper{
+    background-image: url(/images/red-circle.png);
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 100%;
+   
+    display: flex;
+    justify-content: center;
+    padding: 0 50px;
+}
+
+.banner-title{
+    font-weight: 600;
+    font-size: 96px;
+    line-height: 96px;
+    text-transform: uppercase;
+    margin-top: 160px;
+}
+
+.banner-text{
+    max-width: 683px;
+    font-size: 48px;
+    line-height: 60px;
+    margin-top: 10px;
+}
+
+.banner-info_list{
+    margin-top: 75px;
+    display: flex;
+}
+
+.banner-info_item{
+    max-width: 185px;
+    padding-left: 60px;
+    position: relative;
+}
+
+.banner-info_item:not(:last-child){
+    margin-right: 120px;
+}
+
+.banner-info_item_1::before{
+    content: url(/images/banner-1.svg);
+    position: absolute;
+    top: 3px;
+    left: -7px;
+}
+
+.banner-info_item_2::before{
+    content: url(/images/banner-2.svg);
+    position: absolute;
+    top: -3px;
+    left: -10px;
+}
+
+.banner-info_item_3::before{
+    content: url(/images/banner-3.svg);
+    position: absolute;
+    top: -3px;
+    left: -10px;
+}
+
+.banner-items-img{
+    margin-top: 70px;
+}
+
+.banner-img{
+    max-width: 100%;
+}
+
+.banner-items-center{
+    color: #000000;
+    width: 452px;
+    height: 452px;
+    background-color: #ffffff;
+    border-radius: 50%;
+    position: absolute;
+    top: 620px;
+    text-align: center;
+}
+
+
+.banner-center-img-top{
+    margin-top: 22px;
+    max-width: 201px;
+    max-height: 201px;
+}
+
+.banner-center-img-bottom{
+    width: 53px;
+}
+
+.banner-center-title{
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 38px;
+    margin-bottom: 22px;
+}
+
+.banner-center-info{
+    line-height: 20px;
+    margin-bottom: 22px;
+    padding-left: 20px;
+}
+
+.banner-center-info::before{
+    content: url(/images/banner-m.svg);
+    position: absolute;
+    margin-left: -35px;

```

Comment:
better to avoid negative margins using. it's useful in some cases 
but here just `right` or `left` can be used

Reply:
I removed the negative margin and used the negetive left.  Is it correct to use negative values of left and right?